content/200-concepts/100-components/01-prisma-schema/04-data-model.mdx

```diff
@@ -1010,9 +994,72 @@ Support for [`autoincrement()`](../../../reference/api-reference/prisma-schema-r
 
 **Relational database connectors** implement `autoincrement()`, `dbgenerated()`, and `now()` at database level. The **MongoDB connector** does not support `autoincrement()`, and `now()` is implemented at Prisma level. `dbgenerated()` can only be used to generate an `ObjectId`.
 
-## Relations
+## Constraints and indexes in the Prisma schema
 
-Refer to the [relations documentation](relations) for more examples and information about relationships between models.
+Constraints and indexes are represented by [attributes](#defining-attributes) in the Prisma schema. For example:
+
+- `@unique` represents a unique constraint on a single column
+- `@@index` represets an index on one or more columns
+- `@relation` represents a foreign key constraint
+
+Constraints and indexes have **names** in the underlying database. You can [define constraint and index names in the Prisma schema in 2.27.0 and later](#constraint-and-index-names-preview). 
+
+Foreign key constraints - also called referential constraints - also define a referential **action**. You can [define referential actions in the Prisma schema in 2.26.0 and later](#referential-actions-preview).
+
+### Constraint and index names (Preview)
+
+In [2.27.0](https://github.com/prisma/prisma/releases/tag/2.27.0) and later, you can use the `map` field to define the **underlying constraint and index names** in the Prisma schema. The `map` field is _only_ visible in the schema if you use a constraint name that differs from Prisma's naming convention.
+
+Named constraints is a Preview feature. To enable this feature:
+
+1. Add `namedConstraints` to the `previewFeatures` field in the `generator` block:
+
+```prisma
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["namedConstraints"]
+}
+```
```

Comment:
````suggestion
	  ```prisma
	  generator client {
	    provider        = "prisma-client-js"
	    previewFeatures = ["namedConstraints"]
	  }
	  ```
````

Indent so this is in the list